---
title: "Discussion"
order: 5
---

The extreme virulence and environmental stability of _Bacillus anthracis_ requires high-confidence assays that simultaneously enable trace detection and are robust to reporting alarming false positives. Moreover, the specific biology of _B. anthracis_ and other _Bacillus_ species requires an assay that independently characterizes both the core genome and virulence plasmids in the context of complex metagenomic samples. Indeed, _Bacillus cereus_ isolates have been recently found containing the pXO1 and pXO2 virulence plasmids and causing Anthrax-like disease ([Klee, et al., (2010)](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0010986)).

High throughput sequencing offers a promising foundation for such an assay, but the inherent scale and complexity of the resulting datasets tends to confound "one size fits all" analytic approaches (doubly so for metagenomic samples). Indeed, this inquiry was motivated by an alarming false positive arising from the broad application of exploratory metagenomic tools. 

Here, we present a targeted assay for the detection of _Bacillus anthracis_ which is specifically designed with the biology of the pathogen in mind and complements, rather than replaces, unbiased metagenomic analysis. Initiated automatically or at the behest of the end user, the assay detects the diagnostic plcR SNP, the pXO1/pXO2 virulence plasmids, the core _B. anthracis_ genome, and a set of sequences conserved broadly across _Bacillus_ species. 

We tested the detection panel against a diverse set of previously-sequenced _Bacillus_ species – demonstrating its sensitivity across a range of _B. anthracis_ strains, sequencing platforms, sequencing depths, and the presence or absence of confounding near neighbor species. Indeed, all of the 23 analyzed _B. anthracis_ isolates contained 65.2% or more of the core genome markers, while all of the 42 analyzed other _Bacillus_ isolates contained 2.5% or lower core genome markers ([Figure 2](#figure-2)). Moreover, the limit-of-detection analysis shows that this assay was extremely robust to the presence of a near neighbor *Bacillus* ([Figure 3](#figure-3)). 

This assay directly addresses the potentially confounding scenario in which a small number of _B. anthracis_ SNPs are present in a deeply-sequenced metagenomic sample. Those sequences may be present due to trace levels of the pathogen, but they may alternately be caused by a large number of reads from another _Bacillus_ species and a low level of sequencing instrument error. To distinguish between these possibilities, we present the level of evidence for _B. anthracis_ in the context of the relative abndance of all _Bacillus_ species in a sample. This allows an end user to directly compare the relative abundance of _B. anthracis_ to other _Bacillus_ species and more accurately judge whether a given sample merits concern. We also believe that this panel offers a model for the development of easy-to-interpret sequencing-backed assays that serve the needs of applied end users.