---
title: "Discussion"
order: 5
---
The bioinformatic analysis module presented here for the specific detection of _Bacillus anthracis_ is designed with the biology of that pathogen in mind. A single SNP has been identified previously as being contained universally and exclusively among isolates of _B. anthracis_, and that finding is recapitulated by this analysis. The virulence plasmids pXO1 and pXO2 have been detected in rare cases of _Bacillus cereus_ strains causing Anthrax-like symptoms, and therefore merit detection on a basis that is independent from the core genome. The main chromosome of _B. anthracis_ contains many regions that are similar to other species of _Bacillus_, and so a set of 1,605 species-specific marker sequences identified by Petit, et al., (2015) were used as evidence for species-level detection.

However, any method of detection must be calibrated against a biologically relevant set of targets in order to provide analytical bounds for confident detection. In this analysis we used a set of publicly available short-read datasets in order to quantify the range of detection that discriminates between _B. anthracis_ and other _Bacillus_ species. All of the 23 analyzed _B. anthracis_ isolates contained 65.2% or more core genome markers, while all of the 42 analyzed other _Bacillus_ isolates contained 2.5% or lower core genome markers ([Figure 2](#figure-2)).

While the inclusivity/exclusivity testing provides a valuable grounding in the genomic diversity of _B. anthracis_, further testing was performed in order to assess how that genomic diversity interacted with low-abundance input samples. This limit-of-detection analysis showed robust detection that was not impacted by the presence of a near neighbor *Bacillus* ([Figure 3](#figure-3)). We believe that both the presence of potentially-confounding near neighbor organisms, as well as the statistical skew of marker counts, can be used to achieve an extremely high certainty of detection. While it could be argued that a single species-specific read is a sufficient level of evidence for detection, we propose that the detection of organisms whose presence would trigger significant intervention to protect human life, such as _B. anthracis_, require a higher standard of analysis and statistical certainty.

Furthermore, it is rarely the case that a single bioinformatic analysis is sufficient for the operational understanding of a sample or group of samples. Rather, different samples are likely to benefit from different sets of complimentary analyses, such as community composition, phylogenetic clustering, MLST, or functional prediction. Moreover, many analytical tasks are fundamentally interdependent, e.g. this high-confidence detection of _B. anthracis_ may only need to be executed for datasets that meet a lower threshold for _B. anthracis_ detection following read-level metagenomic classification against an exhaustive database of microbial genomes. The One Codex platform provides an ecosystem for all of these analyses to co-exist, presenting users with a graphical display for the set of microbial analyses that is appropriate for each dataset.
