---
title: "Results"
order: 4
---

### Inclusivity & Exclusivity Testing
For each sample, the result of the detection panel was summarized as the proportion of markers detected from the main chromosome. [Figure 2](#figure-2) shows a frequency histogram of the proportion of markers detected for the target organism (*B. anthracis*) and other *Bacilli*. The lowest proportion of core genome markers detected in a *B. anthracis* dataset was 65.2%, and the highest proportion of core genome markers detected in the exclusion dataset was 2.5% (for [SRR2157174](http://www.ncbi.nlm.nih.gov/sra/SRR2157174/)). The plcR SNP was detected in 100% of all *B. anthracis* isolates and in none of the other *Bacillus* isolates.

The proportion of *B. anthracis* reads / *Bacillus* reads varies across the *B. anthracis* genomes sequenced from 1-9%, reflecting varying levels of genomic conservation and sequencing technologies used [Figure 3](#figure-3).

### Limit of Detection Analysis
Analysis of *in silico* mixed samples containing *B. anthracis* or other *Bacillus* at a range of low abundances is shown in [Figure 4](#figure-4). The proportion of detected markers increases roughly linearly with the amount of target genome coverage in the input dataset. Even at the lowest input level of 0.01X spike-in, the *B. anthracis* score indicates high confidence of detection, while none of the other *Bacillus* spike-in samples show any similar scores. The plcR SNP was detected in 33.3% of samples with a spike-in level of 0.1X and above, 50% of samples at 0.75X, and 83.3% of samples at 2X spike-in.
